src/main/webapp/transcript.js

```diff
@@ -102,3 +104,14 @@ function appendParagraphToContainer(text, container, classes = []) {
 function deleteTranscript() {
   fetch('/delete-transcript', {method: 'POST'});
 }
+
+/**
+ * Coverts a Date representing a video {@code timestamp} into a string.
+ */
+// TODO: Move function to timestamp.js
+function timestampToString(timestamp) {
+  const ts = new Date(timestamp);
```

Comment:
Please do not use abbreviations. Also this is a date, not a timestamp, so a better name would be date.

"Use the shortest possible name, but not at the cost of clarity. Avoid acronyms, abbreviations, and in-team jargon."

Please look at: https://docs.google.com/document/d/1_ldClEZLsI1cMpQl_BfeIlkjTsE2IiJ2PgvhABfO6vs/edit

Reply:
Thanks for the info!
This was a patch from an different pull request that I added in to ensure this pull request compiles. 

I believe it is now fixed and added to timestamp.js instead.

Reply:
I see, I would recommend not doing. If you absolutely must please add a review comment indicating this is not your code or else I end up reviewing the code twice.

Reply:
Will do, sorry about that!